python/Dockerfile.nopandas

```diff
@@ -0,0 +1,7 @@
+ARG PYTHON_VERSION=3.6
```

Comment:
Please add a license header: `NOT APPROVED: python/Dockerfile.nopandas (apache-arrow/python/Dockerfile.nopandas): false`